src/Header/style.scss

```diff
@@ -14,6 +14,7 @@
       margin: 13px 0 15px 40px;
       background: url('../img/logo.svg');
       background-repeat: no-repeat;
+      background-size: contain;
```

Comment:
The image dimensions are different, so we can just container the image within it's parent container